src/components/layout/Page/index.module.scss

```diff
@@ -13,6 +13,13 @@
   }
 }
 
+@media only screen and (min-width: #{$screenLg}px) {
```

Comment:
I can't understand this change. The associated commit message says `Reduces padding for medium screen sizes` but this media query will only work for large sizes. Am I missing something?

Reply:
so the query that will work for xs, sm and md sizes is just above this one:
```
@media only screen and (max-width: #{$screenLg}px) {
```

Reply:
Yes sorry, my bad writing the commit message. Basically that change it's for larger screen sizes (1200 and above)